_targets.R

```diff
@@ -5,45 +5,79 @@ library(tibble)
 options(tidyverse.quiet = TRUE)
 library(tidyverse)
 
-tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate"))
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr",
+                            "rnaturalearth", "cowplot", "lubridate",
+                            "leaflet", "leafpop", "htmlwidgets"))
 
 # Load functions needed by targets below
 source("1_fetch/src/find_oldest_sites.R")
 source("1_fetch/src/get_site_data.R")
 source("2_process/src/tally_site_obs.R")
+source("2_process/src/summarize_targets.R")
 source("3_visualize/src/map_sites.R")
+source("3_visualize/src/plot_data_coverage.R")
 source("3_visualize/src/plot_site_data.R")
+source("3_visualize/src/map_timeseries.R")
 
 # Configuration
-states <- c('WI','MN','MI','IL','IN','IA')
+states <- c('WI','MN','MI','IL','IN','IA','OH')
 parameter <- c('00060')
 
+mapped_by_state_targets <- tar_map(
+  tibble(state_abb = states) %>%
+    mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
+  tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
+                                           state_cd == state_abb)),
+  tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
+  # Insert step for tallying data here
+  tar_target(tally, tally_site_obs(nwis_data)),
+  # Insert step for plotting data here
+  tar_target(timeseries_png,
+             plot_site_data(state_plot_files,
+                            nwis_data,
+                            parameter),
+             format = 'file'),
+  names = state_abb
+)
+
 # Targets
 list(
   # Identify oldest sites
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
-
-  tar_map(
-    tibble(state_abb = states) %>%
-      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
-    tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
-                                             state_cd == state_abb)),
-    tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
-    # Insert step for tallying data here
-    tar_target(tally, tally_site_obs(nwis_data)),
-    # Insert step for plotting data here
-    tar_target(timeseries_png,
-               plot_site_data(state_plot_files,
-                              nwis_data,
-                              parameter),
-               format = 'file'),
-    names = state_abb
+  # Reference/define the split targets
+  mapped_by_state_targets,
+  # Combine the split targets tallies
+  tar_combine(
+    obs_tallies,
+    mapped_by_state_targets[[3]],
```

Comment:
With `unlist = FALSE` added to your `tar_map()` call (see above), then you can easily refer to the nested targets within your `tar_map()` call in a clear way. E.g. you can just use the target name to refer to them. So, this becomes `mapped_by_state_targets$tally` instead of `mapped_by_state_targets[[3]]`. This protects you a bit more from potential reordering mishaps :)